### PR TITLE
Fix the Integration UI Test cases

### PR DESCRIPTION
**Pull request by sajithab**
jira - https://wso2.org/jira/browse/APPM-538

### CODE REVIEW

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java [hunk 1]
```diff
@@ -337,7 +336,46 @@ private HttpResponse addRole(String roles, String appId) throws Exception {
...
+	 * @param throttlingTier
+	 * @param objPartialMappings
+	 * @param policyGroupDesc
+	 * @return policyGroupId
+	 * @throws Exception
+	 */
+	public String savePolicyGroup(String policyPartial, String anonymousAccessToUrlPattern, String policyGroupName,
+								  String throttlingTier, String objPartialMappings, String policyGroupDesc) throws Exception {
+
+		String payLoad = "policyPartial="+ URLEncoder.encode(policyPartial,"UTF-8") +
```
Comment (dinusha-dilrukshi) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java]: Use StringBuilder here.

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java [hunk 2]
```diff
@@ -337,7 +336,46 @@ private HttpResponse addRole(String roles, String appId) throws Exception {
...
+
+		HttpResponse response =
+				HttpRequestUtil.doPost(new URL(backEndUrl +
+								"/publisher/api/entitlement/policy/partial/policyGroup/save"), payLoad,
+						requestHeaders);
+		if (response.getResponseCode() == 200) {
+			JSONObject saveObject = new JSONObject(response.getData());
+			String policyGroupId = saveObject.optJSONObject("response").getString("id");
+			return policyGroupId;
+		} else {
```
Comment (dinusha-dilrukshi) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java]: Remove System.out and use log.info.

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java [hunk 3]
```diff
@@ -20,6 +21,13 @@
     private String overview_trackingCode = "AM_-1305386558";
     private String roles = "";
     private String tags = "";
+    private String uritemplate_policyGroupIds;
```
Comment (lahiru823) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java]: Line24-30 : use proper formatting

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java [hunk 4]
```diff
@@ -20,6 +21,13 @@
...
+    private String uritemplate_javaPolicyIds;
+    private String uritemplate_policygroupid0="1";
+    private String uritemplate_policygroupid1="1";
+    private String uritemplate_policygroupid2="1";
+    private String uritemplate_policygroupid3="1";
+    private String uritemplate_policygroupid4="1";
```
Comment (lahiru823) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java]: Line 26-30 : use proper formatting

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/TestUtils/ApplicationInitializingUtil.java [hunk 5]
```diff
@@ -100,9 +112,12 @@ public void testApplicationCreation(String prefix) throws Exception {
             policyPartialId = responseId.getString("id");
         }
 
+        String policyGropuId = appMPublisher.savePolicyGroup(xml, anonymousAccessToUrlPattern, policyGroupName,
+                throttlingTier, objPartialMappings, policyGroupDesc);
+
         int hostPort = 8080;
-        AppCreateRequest appCreateRequest = createSingleApp(appName+prefix, version, transport, appURL, hostPort,
-                appProp.getTier(), policyPartialId);
+        AppCreateRequest appCreateRequest = createSingleApp(appName+prefix, appDisplayName, version, transport, appURL, hostPort,
```
Comment (lahiru823) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/TestUtils/ApplicationInitializingUtil.java]: Line 119 : use proper formatting

File: modules/integration/tests-ui/src/test/resources/appconfig.properties [hunk 6]
```diff
@@ -3,9 +3,17 @@ appVersion = 1.0.0
...
 tier = Bronze
 tags = youtubeApi
 role = internal/reviewer
 path = /sample
 appSample = ./samples/sample.war
-policyFile = ./samples/policy.xml
\ No newline at end of file
+policyFile = ./samples/policy.xml
+policyGroupName = test
+policyGroupDesc =null
```
Comment (lahiru823) [inline modules/integration/tests-ui/src/test/resources/appconfig.properties]: Line 14 : use proper formatting

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java (hunk 1 above)
Comment (sajithab) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java]: This code segment has been refactored and this code line is not in here any more.

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java (hunk 2 above)
Comment (sajithab) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/APPMPublisherRestClient.java]: This code segment has been refactored and this code line is not in here any more.

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java (hunk 4 above)
Comment (sajithab) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java]: This code segment has been refactored and this code line is not in here any more.

File: modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/TestUtils/ApplicationInitializingUtil.java (hunk 5 above)
Comment (sajithab) [inline modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/TestUtils/ApplicationInitializingUtil.java]: This code segment has been refactored and this code line is not in here any more.

File: modules/integration/tests-ui/src/test/resources/appconfig.properties (hunk 6 above)
Comment (sajithab) [inline modules/integration/tests-ui/src/test/resources/appconfig.properties]: This code segment has been refactored and this code line is not in here any more.

**Pull request merged**